lib/sycamore/sycamore/transforms/text_extraction/pdf_miner.py

```diff
@@ -24,31 +24,22 @@ def __init__(self):
         from pdfminer.layout import LAParams
         from pdfminer.pdfinterp import PDFPageInterpreter, PDFResourceManager
 
-        rm = PDFResourceManager()
-        param = LAParams()
-        self.device = PDFPageAggregator(rm, laparams=param)
-        self.interpreter = PDFPageInterpreter(rm, self.device)
+        self.rm = PDFResourceManager()
+        self.param = LAParams()
```

Comment:
Do we need to store these two in `self`?

Reply:
removed.